mozjs/mozjs/js/src/vm/StructuredClone.cpp

```diff
@@ -1894,13 +1894,13 @@ bool JSStructuredCloneWriter::traverseError(HandleObject obj) {
   // Non-standard: Serialize |stack|.
   // The Error stack property is saved as SavedFrames.
   RootedValue stack(cx, NullValue());
-  RootedObject stackObj(cx, unwrapped->stack());
+  /*RootedObject stackObj(cx, unwrapped->stack());
   if (stackObj && stackObj->canUnwrapAs<SavedFrame>()) {
     stack.setObject(*stackObj);
     if (!cx->compartment()->wrap(cx, &stack)) {
       return false;
     }
-  }
```

Comment:
If this is non-standard, why is it working for Gecko I wonder?

Reply:
Done!